src/EnergyPlus/SingleDuct.cc

```diff
@@ -1285,24 +1312,32 @@ namespace SingleDuct {
             // Setup output for the air terminal single duct constant volume no reheat.
             SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
                                 OutputProcessor::Unit::m3_s,
-                                Sys(SysNum).OutdoorAirFlowRate,
+                                sd_airterminal(SysNum).OutdoorAirFlowRate,
                                 "System",
                                 "Average",
-                                Sys(SysNum).SysName);
+                                sd_airterminal(SysNum).SysName);
+            SetupOutputVariable(
+                "Zone Air Terminal Sensible Heating Energy", OutputProcessor::Unit::J, sd_airterminal(SysNum).HeatEnergy, "System", "Sum", sd_airterminal(SysNum).SysName);
+            SetupOutputVariable(
+                "Zone Air Terminal Sensible Cooling Energy", OutputProcessor::Unit::J, sd_airterminal(SysNum).CoolEnergy, "System", "Sum", sd_airterminal(SysNum).SysName);
+            SetupOutputVariable(
+                "Zone Air Terminal Sensible Heating Rate", OutputProcessor::Unit::W, sd_airterminal(SysNum).HeatRate, "System", "Average", sd_airterminal(SysNum).SysName);
+            SetupOutputVariable(
+                "Zone Air Terminal Sensible Cooling Rate", OutputProcessor::Unit::W, sd_airterminal(SysNum).CoolRate, "System", "Average", sd_airterminal(SysNum).SysName);
```

Comment:
@Nigusse These were removed because they duplicate the same outputs that are already set up by the air distribution unit [here](https://github.com/NREL/EnergyPlus/blob/55f5f74c4f7b5b99e0110e5c7a158fd85bb20cc2/src/EnergyPlus/ZoneAirLoopEquipmentManager.cc#L470-L495).

Reply:
@mjwitte Thanks. I correct in this branch and push again.